types/plotly.js/index.d.ts

```diff
@@ -651,6 +651,11 @@ export interface PlotData {
 		x: number[];
 		y: number[];
 	}>;
+	title: Partial<{
```

Comment:
Is there a reason you don't define a base type to do `Partial<MyType>` instead (as done above: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42124/files#diff-33d497cc65e43b9280207de4e7e1d9d5R621)?

Reply:
@jessetrinity no particular reason. I made the change you requested.

Reply:
Cool - just wanted to make sure I wasn't missing some cool type magic.